### PR TITLE
Fix the broken image path in get_started

### PR DESCRIPTION
**Pull request by hurutoriya**
I fixed the broken image path.

### CODE REVIEW

**Comment by hurutoriya** [comment]
Hello @chuanyu  😄

**Comment by chuanyu** [comment]
Thanks for the fix!

File: g3doc/get_started.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ TFMA can perform large-scale distributed evaluation of your model by using
 evaluation results can be visualized in a Jupyter notebook using the frontend
 components included in TFMA.
 
-![TFMA Slicing Metrics Browser](/images/tfma-slicing-metrics-browser.png)
+![TFMA Slicing Metrics Browser](/g3doc/images/tfma-slicing-metrics-browser.png)
```
Comment (chuanyu) [inline g3doc/get_started.md]: How about using a relative path: `images/tfma-slicing-metrics-browser.png` instead?

Comment (hurutoriya) [reply g3doc/get_started.md]: I see, I'll fix. thank you! 😄

**Comment by hurutoriya** [comment]
@chuanyu Thank you for Nice feedback. I fix to a relative path. 💪

**Pull request merged**